bin/debug.ml

```diff
@@ -0,0 +1,319 @@
+open Stdune
+open Import
+
+let doc =
+  "Debug a bytecode program in a similar environment as if installation was \
```

Comment:
can you move this to `dune ocaml debug`?

Reply:
Yes, this will be split into a separate PR. However, I'd like to keep the computation of the forward and inverse maps in a single PR so we can do consistency checking of them.